Bindings/Java/OpenSimJNI/CMakeLists.txt

```diff
@@ -30,21 +32,44 @@ add_custom_command(
             "${CMAKE_SOURCE_DIR}/Bindings/OpenSimHeaders_actuators.h"
             "${CMAKE_SOURCE_DIR}/Bindings/OpenSimHeaders_analyses.h"
             "${CMAKE_SOURCE_DIR}/Bindings/OpenSimHeaders_tools.h"
-    COMMENT "Generating java bindings source code with SWIG."
```

Comment:
I'd like for this to be cleaned up but it's also fine if it's left as-is. We should revisit the handling of dependencies anyway, since as @klshrinidhi has noted, this list of dependencies is incomplete. So it's fine if we just leave this to be dealt with when we address #790.